buildh/core.py

```diff
@@ -382,7 +382,7 @@ def make_dic_lipids_with_indexes(universe_woH, dic_lipid, dic_OP):
     dic_lipids_with_indexes = {}
     for Cname in dic_lipid.keys():
         if Cname in carbons2keep:
-            dic_lipids_with_indexes[Cname] = dic_lipid[Cname]
+            dic_lipids_with_indexes[Cname] = dic_lipid[Cname].copy()
```

Comment:
Just to understand, why do we need to a deep copy here?

Reply:
you're too fast, I was just written this specific modification in my reply below :)

Reply:
Haha sorry 🤣